akka-docs/src/test/scala/docs/stream/operators/sourceorflow/ScanAsync.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sourceorflow
+import akka.stream.scaladsl.Source
+
+import scala.concurrent.Future
+
+object ScanAsync {
+  def scanAsyncExample(): Unit = {
+    import akka.actor.ActorSystem
+
+    implicit val system: ActorSystem = ActorSystem()
+
+    //#scanAsync
+    //Use system's dispatcher as ExecutionContext
```

Comment:
`as ExecutionContext for the Future` so that it doesn't seem like it is needed for `scanAsync` itself.

Or just use `Future.successful(value)` so that it works the same way as the Java sample.